src/python/shorah_dec.py

```diff
@@ -25,6 +25,11 @@
     file of corrected reads
 '''
 
+from __future__ import print_function
+from __future__ import division
+from builtins import dict
+from future.utils import listitems
+from builtins import range
```

Comment:
Autoconf doesn't check dependencies and doesn't care at build time. conda also won't be a problem, because we will package it as a py3 package. Only those users wishing to use py2 need to use future then. I guess that is a good solution?